src/RadixDlt.NetworkGateway.PostgresIntegration/Models/ValidatorEmissionHistory.cs

```diff
@@ -67,8 +67,8 @@
 
 namespace RadixDlt.NetworkGateway.PostgresIntegration.Models;
 
-[Table("validator_emission_statistics")]
-internal class ValidatorEmissionStatistics
+[Table("validator_emission_history")]
+internal class ValidatorEmissionHistory
```

Comment:
is that table used anywhere after introducing cumulative table with sum for each epoch?

Reply:
No, it is not. I've left it to reduce the impact of the change (backwards compatibility).

Reply:
i'd vote for removing it but I'm aware we don't have any tool to check if anyone is using that.

Reply:
OK, dropped.